cmd/nitro/nitro.go

```diff
@@ -826,14 +826,19 @@ func applyChainParameters(ctx context.Context, k *koanf.Koanf, chainId uint64, c
 		chainDefaults["init.empty"] = true
 	}
 	if parentChainIsArbitrum {
-		l2MaxTxSize := gethexec.DefaultSequencerConfig.MaxTxDataSize
-		bufferSpace := 5000
-		if l2MaxTxSize < bufferSpace*2 {
-			return false, fmt.Errorf("not enough room in parent chain max tx size %v for bufferSpace %v * 2", l2MaxTxSize, bufferSpace)
-		}
-		safeBatchSize := l2MaxTxSize - bufferSpace
-		chainDefaults["node.batch-poster.max-size"] = safeBatchSize
-		chainDefaults["node.sequencer.max-tx-data-size"] = safeBatchSize - bufferSpace
+		if chainInfo.DasIndexUrl != "" {
+			chainDefaults["node.batch-poster.max-size"] = 1000000
+			chainDefaults["node.sequencer.max-tx-data-size"] = 995000
```

Comment:
The sequencer MaxTxDataSize shouldn't be increased for DAS chains, in case the chain needs to fall back to on-chain posting. It should still be decreased if it's an orbit chain.

Also, the batch poster MaxSize should be increased even if `!parentChainIsArbitrum`. I'd put it as its own check after this one to overwrite it.